build.gradle

```diff
@@ -13,23 +13,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import kotlin.KotlinVersion
-
 buildscript {
   dependencies {
     // Needs to go first, else gradle's forced 1.3.72 version takes precedence
-    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.10")
+    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.20")
```

Comment:
There is Kotlin 1.4.21

Reply:
https://github.com/ZacSweers/ticktock/pull/46/commits/dd1d450a491aeee13411a375d60c535f340b8aab